spec/system/school/customization_spec.rb

```diff
@@ -209,6 +225,59 @@ def image_path(filename)
 
   scenario 'user who is not logged in gets redirected to sign in page' do
     visit customize_school_path
-    expect(page).to have_text("Please sign in to continue.")
+    expect(page).to have_text('Please sign in to continue.')
+  end
+
+  scenario 'school admin rearrange and edit link title, url' do
+    # sign in user
+    sign_in_user school_admin.user, referrer: customize_school_path
+    expect(page).to have_content('You can customize links on the header.')
+
+    # Add four links and check that they're all displayed.
+    find('button[title="Edit header links"]').click
+
+    (1..4).each do |link_number|
+      fill_in 'Title', with: "Test Link #{link_number}"
+      fill_in 'Full URL', with: "http://example.com/#{link_number}"
+      click_button 'Add a New Link'
+      expect(page).to have_selector(
+        "button[title='Delete http://example.com/#{link_number}']"
```

Comment:
Looks like the `title` attribute here is just added for testing purpose. Having the full URL here in the attribute doesn't look clean. 

Do check implementation and usages of `Spread` component where we can keep such data in custom data attributes `school-link-id` and use that to identify elements in the tests.